var/spack/repos/builtin/packages/paraview/package.py

```diff
@@ -25,8 +27,8 @@ class Paraview(Package):
 
     depends_on('bzip2')
     depends_on('freetype')
-    depends_on('hdf5')
     depends_on('hdf5+mpi', when='+mpi')
+    depends_on('hdf5~mpi', when='~mpi')
```

Comment:
I've seen including MPI in non-MPI builds of things cause some problems so I think being safe is ok here.  We can try to figure out why it broke and constrain it less later if we want.